kt/kt_metrics.go

```diff
@@ -42,6 +42,14 @@ func NewTrackedConn(host string, port int, poolsize int, timeout time.Duration,
 		opTimer: opTimer}, nil
 }
 
+// NewTrackedConnFromConn returns a tracked connection that simply wraps the given
+// database connection.
+func NewTrackedConnFromConn(conn *Conn, opTimer *prometheus.SummaryVec) (*TrackedConn, error) {
```

Comment:
I kept the error argument here to keep an identical signature to NewTrackedConn, and we might need to introduce error conditions in the future...?